docs/tutorial.rst

```diff
@@ -910,6 +910,10 @@ By default, the following variables are available for use:
   * ``user``: The username of the current user
   * ``hostname``: The name of the host computer
   * ``cwd``: The current working directory
+  * ``dir_cwd``: The dirname of the current working directory, e.g. ``/path/to`` in
```

Comment:
For some reason, these names feel strange to me.  Can we call these `cwd_dir` and `cwd_base`?

Even then, `cwd_dir` feels weird since `cwd` is, itself, a directory...